CODEOWNERS

```diff
@@ -1 +0,0 @@
-* @mvadari @ledhed2222
```

Comment:
shouldn't we replace with new owners? if none defined, doesn't seem bad to me to keep this until there are new ones defined

Reply:
We don't have any codeowners in the xrpl.js repo and it's fine - IMO this isn't really necessary for these repos.

Reply:
Unless you think it'll be actually useful given that different groups are going to be independently working in these repos? Either way, the decision should be the same across the two repos we control(led).